README.md

```diff
@@ -35,59 +39,705 @@ var auth0 = require('auth0')({
 });
 ~~~
 
-### User's metadata
```

Comment:
All user_metadata and app_metadata doc and sample code was removed from the readme? it would keep it